### DISCUSSION TITLE
On the windows10 platform

### DISCUSSION THREAD

**Discussion by sunmamba** [danielegrattarola/spektral#261] [Q&A] [answered]
I have successfully installed spektral on the win10 platform, can I use it for training?
thanks!

**Comment by ghost** [comment, marked as answer]
Yes. You can use spektral on Windows 10 machines. You may test the training success via running [official examples](https://graphneural.network/examples/) of the spektral usage.

**Comment by danielegrattarola** [comment]
Just to add to @Matin-Macktoobian's reply: I've just officially added Windows as a supported platform, given that I was able to run all unit tests also on Windows :)
So yeah, all the documentation and examples should work also on Windows, give it a try
